etc/logrotate.d.d/zulip-docker

```diff
@@ -0,0 +1,29 @@
+docker exec -it zulip bash -c 'cat > /etc/logrotate.d/zulip-docker' << 'EOF'
+/var/log/zulip/*.log {
+    daily
+    rotate 7
+    compress
+    delaycompress
+    notifempty
+    missingok
+    create 644 zulip zulip
+    sharedscripts
+    postrotate
+        supervisorctl reopen-logs
+    endscript
+}
+
+/var/log/supervisor/*.log {
+    daily
+    rotate 7
+    compress
+    delaycompress
+    notifempty
+    missingok
+    create 644 root root
+    sharedscripts
+    postrotate
+        supervisorctl reopen-logs
+    endscript
+}
+EOF
```

Comment:
Zulip ships with a logrotate configuration file; I don't think we should need to provide this in the container image as well, as `zulip` itself should have installed it already. Would be good to avoid duplicating, as the copy here will get stale.